gatsby-site/src/components/View/index.js

```diff
@@ -12,12 +12,9 @@ const View = (props) => {
       <Illustration />
       <ReactNavbar />
       <Jumbotron>
-        <div id="title">
           <h1 className="display-3">Building <br></br> better software, together.</h1>
```

Comment:
No need for <br></br> now that you fixed the word-breaks in body. Also, the word "focus" in the lead statement is mispelled.  These are really small things that can go in your next PR.